src/llmq/quorums_signing.cpp

```diff
@@ -276,6 +279,16 @@ void CRecoveredSigsDb::RemoveRecoveredSig(CDBBatch& batch, Consensus::LLMQType l
     batch.Erase(k3);
     batch.Erase(k4);
 
+    if (deleteTimeKey) {
+        CDataStream writeTimeDs(SER_DISK, CLIENT_VERSION);
+        if (db.ReadDataStream(k2, writeTimeDs) && writeTimeDs.size() == sizeof(uint32_t)) {
```

Comment:
should this check `writeTimeDs.size() == sizeof(uint32_t)` be removed in a future version?

Reply:
I didn't implement any upgrade logic, so if we ever remove this, then very very far in the future

Reply:
I don't follow... I thought this check differentiated between the old version (1 byte) and the new version (4 bytes). Under that, once the version after 0.14.1 is deployed I would think the old version would no longer be present and because of that you wouldn't need to do the check

Reply:
True, but when a user then later skips 14.1 and directly goes from 14.0 to 14.2, he'll end up with a crash. I'm for removing this check, but in multiple versions in the future.

Reply:
Okay, I would want to put in a TODO so that we don't forget to remove at some point

Reply:
Done.